cfme/tests/test_login.py

```diff
@@ -12,6 +12,35 @@
 pytestmark = pytest.mark.usefixtures('browser')
 
 
+def new_cred(username, password):
+    username = username or conf.credentials['default']['username']
+    password = password or conf.credentials['default']['password']
+    cred = Credential(principal=username, secret=password)
+    return cred
+
+
+def new_user_instantiate(appliance, username=None, password=None, name=None):
+    cred = new_cred(username=username, password=password)
+    name = name or 'Administrator'
```

Comment:
Please use fauxfactory to get a randomized name so that subsequent test runs don't fail just because the appliance wasn't completely cleaned up.